Tar command fails with beta 5

```
Downloading theme "bootstrap" release "8.x-3.0-rc2"


 [ERROR] Unable to execute the following command '/bin/tar' '--extract' '--file=/tmp/bootstrap.tar.gz' '--overwrite-dir'
 '--overwrite' '--gzip' '--directory' '/OMITTED/themes' {output: /bin/tar: '--overwrite' cannot 
 be used with '--overwrite-dir'                                                                                         
 Try '/bin/tar --help' or '/bin/tar --usage' for more information.                                                      
 }

```

Debian Testing
PHP 7
Drupal 8.1.x

Seems like a very simple usage bug in the source code.

Edit:
Using the phar as well.

Same here , 
```
sargath ~/public_html $ drupal site:new new-project --latest

 Downloading "drupal" "8.1.8"

                                                                                                                        
 [ERROR] Unable to execute the following command '/usr/bin/tar' '--extract' '--file=/tmp/drupal.tar.gz'
```
```                                                                                                                        
 Drupal "8.1.8" was downloaded in directory "drupal-8.1.8"                                                              
                                                                                                                        

                                                                                                                        
 [ERROR] An error occurred while renaming directory as "new-project"                                                            
                                                                                                                        

sargath ~/public_html $ php -v
PHP 7.0.9 (cli) (built: Jul 20 2016 17:12:28) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies
    with Xdebug v2.4.1, Copyright (c) 2002-2016, by Derick Rethans

sargath ~/public_html $ drupal --version
"Drupal Console" version "1.0.0-beta5"
```

--- a/composer.json
+++ b/composer.json
@@ -38,7 +38,7 @@
     "require": {
         "php": "^5.5.9 || ^7.0",
         "drupal/console-core" : "dev-master",
-        "alchemy/zippy": "0.3.5",
+        "alchemy/zippy": "0.4.3",
         "composer/installers": "~1.0",
         "symfony/css-selector": ">=2.7 <3.2",
         "symfony/dom-crawler": ">=2.7 <3.2",

